lib/controllers/issueish-detail-controller.js

```diff
@@ -15,7 +16,8 @@ export class BareIssueishDetailController extends React.Component {
       owner: PropTypes.shape({
         login: PropTypes.string.isRequired,
       }).isRequired,
-      issueish: PropTypes.any, // FIXME from IssueishPaneItemContainer.propTypes
+      pullRequest: PropTypes.any, // FIXME
+      issue: PropTypes.any, // FIXME from IssueishPaneItemContainer.propTypes
```

Comment:
We can probably just delete these `FIXME` comments honestly 🙈 I don't think it's ever going to be worth it to fully spec out the Relay responses in PropTypes. My fingers hurt even thinking about it.